esp/platform/espcfg.cpp

```diff
@@ -111,11 +111,62 @@ StringBuffer &CVSBuildToEspVersion(char const * tag, StringBuffer & out)
     return out;
 }
 
+int CSessionCleaner::run()
+{
+    try
+    {
+        PROGLOG("CSessionCleaner Thread started.");
+
+        VStringBuffer xpath("%s*", PathSessionSession);
+        int checkSessionTimeoutMillSeconds = checkSessionTimeoutSeconds * 60;
```

Comment:
minor point, this variable isn't really necessary

Reply:
But, I do not want to calculate it in every loops.